modules/configuring-aws-iam-for-cross-account-access.adoc

```diff
@@ -0,0 +1,78 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-10-19
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-aws-iam-for-cross-account-access_{context}"]
+= Configuring AWS IAM for cross-account access
+
+You can configure {aws-short} Identity Access Management (IAM) for cross-account access.
+
+.Prerequisites
+* You have administrator permissions.
+
+.Procedure
+
+. Create an initial role in your cluster account.
```

Comment:
grant your initial account( either a user or  an iam role)  permission to  assume the target role

Reply:
What one is better "initial account" or "initial role" or "initial principle" or "initial user"?   As it can be access key or role,  "initial role" make us confuse.

Reply:
Used "initial account"